posts/virtual_environments.md

```diff
@@ -0,0 +1,126 @@
+# Virtual Environments for Data
+*by [Iaroslav Zeigerman](https://github.com/izeigerman), [Ryan Eakman](https://github.com/eakmanrq) and [Toby Mao](https://github.com/tobymao)*
+
+The existing approaches to managing development environments for data are any or all of the following:
+- **Inefficient** - creating new environments is usually associated with additional compute and storage costs. Deploying changes to production after they’ve been tested involves more overhead on top of that.
+- **Non-scalable** - managing multiple environments and concurrent changes is a manual process that is time-consuming and error-prone, especially in bigger organizations.
+- **Unsafe** - there are no guarantees that results observed during development are reproducible on production, especially when deploying changes concurrently.
+
+
+In this post we introduce a novel approach that provides low-cost, efficient, scalable and safe data environments that are easy to use and manage. This will significantly boost the productivity of anyone who has to create and maintain data pipelines.
```

Comment:
```suggestion
In this post we introduce a novel approach that provides low-cost, efficient, scalable, and safe data environments that are easy to use and manage. This will significantly boost the productivity of anyone who has to create and maintain data pipelines.
```